tests/test_analytics.py

```
@@ -34,50 +34,60 @@ def period_analytics(self, days, limit):
+
+def test_maps_activity_filters_to_application_service(analytics_client) -> None:
+    client, manager = analytics_client
+    response = client.get(
+        "/api/analytics/activity?kind=deaths&player=VonCrush&source=structured&search=zombie&days=7&page=2&page_size=20"
+    )
+    assert response.status_code == 200
+    assert manager.arguments == ("deaths", "VonCrush", "structured", "zombie", 7, 2, 20)
+
+
+def test_rejects_non_numeric_pagination(analytics_client) -> None:
+    client, manager = analytics_client
```

Ruff (0.16.1)

[warning] Unpacked variable `manager` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)